Source/Diagnostics/BTDiagnostics.cpp

```diff
@@ -1036,7 +1036,7 @@ BTDiagnostics::InitializeParticleBuffer ()
         for (int isp = 0; isp < m_particles_buffer[i].size(); ++isp) {
             m_totalParticles_flushed_already[i][isp] = 0;
             m_totalParticles_in_buffer[i][isp] = 0;
-            m_particles_buffer[i][isp] = std::make_unique<PinnedMemoryParticleContainer>(&WarpX::GetInstance());
+            m_particles_buffer[i][isp] = std::make_unique<PinnedMemoryParticleContainer>(WarpX::GetInstance().GetParGDB());
```

Comment:
Now that `PinnedMemoryParticleContainer` is of type `WarpXParticleContainer::ContainerLike<amrex::PinnedArenaAllocator>` instead of `amrex::AmrParticleContainer`, its constructor cannot take an `AmrCore object as input. 
This is why we are passed `GetParGDB` instead here.